Assets/Scripts/Level/TutorialTrigger.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections;
+using UnityEngine;
+
+namespace Assets.Scripts.Level
+{
+    /**
+     * Create an instance of TutorialTriggerGroups to populate each Trigger to speed up GetComponent calls
+     */
+    public class TutorialTrigger : MonoBehaviour
+    {
+        public int requiredDeaths;
+        public string[] TutorialText;
+        private PlayerProgress playerProgress;
+        private InGameUI gameUI;
+
+        private int initialDeaths;
+        private bool activated;
+
+        // Use this for initialization
+        void Start()
+        {
+            Collider myCollider = GetComponent<Collider>();
+            myCollider.isTrigger = true;
+            initialDeaths = -1;
+            activated = false;
+        }
+
+        public void AddGameUI(InGameUI ui)
+        {
+            gameUI = ui;
+        }
+
+        public void AddPlayerProgress(PlayerProgress player)
+        {
+            playerProgress = player;
+        }
+
+        private void OnTriggerEnter(Collider other)
+        {
+            if (other.gameObject.layer == PlayerConstants.PlayerLayer)
+            {
+                if (activated) return;

```

Comment:
probably need to remember to deactivate all of them if the player resets the level ;)